### ISSUE TITLE
commons - harmonize spring versions

### ISSUE THREAD

**Issue by pmauduit** [georchestra/georchestra#1329]
`georchestra-commons` does not specify a version for Spring, hence uses the one defined into the root pom. this is not a problem, but the spring version used is fairly old:

```
[INFO] +- org.springframework:spring-webmvc:jar:2.5.6.SEC01:compile
[INFO] |  +- commons-logging:commons-logging:jar:1.1.1:compile
[INFO] |  +- org.springframework:spring-beans:jar:2.5.6.SEC01:compile
[INFO] |  +- org.springframework:spring-context:jar:2.5.6.SEC01:compile
[INFO] |  |  \- aopalliance:aopalliance:jar:1.0:compile
[INFO] |  +- org.springframework:spring-context-support:jar:2.5.6.SEC01:compile
[INFO] |  +- org.springframework:spring-core:jar:2.5.6.SEC01:compile
[INFO] |  \- org.springframework:spring-web:jar:2.5.6.SEC01:compile
```

We should consider changing the default spring version to a newer one, and/or harmonize on every georchestra modules that currently uses the variable.

**Comment by pmauduit** [comment]
Based on the current state on master branche:

- analytics uses 4.2.3.RELEASE
- cas uses 3.2.6-RELEASE (we do not have a hand on this, since the sources comes from upstream)
- catalogapp 3.1.0.RELEASE
- downloadform 3.1.0.RELEASE
- extractorapp 3.1.0.RELEASE
- header 3.1.0.RELEASE
- ldapadmin 4.2.3.RELEASE
- mapfishapp 3.1.0.RELEASE
- security-proxy 3.0.7.RELEASE

**Comment by severo** [comment]
Current state:

| module | Spring version |
| --- | --- |
| analytics | 4.2.3.RELEASE |
| atlas |  4.1.8.RELEASE |
| cas-server-webapp | 3.2.6.RELEASE |
| console | 4.2.3.RELEASE |
| extractorapp | 3.1.0.RELEASE |
| header | 3.1.0.RELEASE |
| mapfishapp | 3.1.0.RELEASE |
| security-proxy | 4.3.13.RELEASE |
| root | 2.5.6.SEC01 |

**Comment by groldan** [comment]
Closing, fixed by #2405
